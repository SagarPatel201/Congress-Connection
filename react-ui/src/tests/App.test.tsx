import React from 'react';
//simport { shallow } from 'enzyme';
//import App from "../App";


/*describe("MyComponent", () => {
  it("should render the App component", () => {
    const wrapper = shallow(<App />);
    expect(wrapper.getElements()).toMatchSnapshot();
  });
});
*/