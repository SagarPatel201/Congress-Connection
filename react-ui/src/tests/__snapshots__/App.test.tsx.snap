// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MyComponent should render my component 1`] = `
Array [
  <div>
    <Homepage />
  </div>,
]
`;

exports[`MyComponent should render the App component 1`] = `
Array [
  <div>
    <Homepage />
  </div>,
]
`;
